packages/components/src/listBox/src/ListBox.module.css

```diff
@@ -0,0 +1,127 @@
+.hop-ListBox  {
+    /* Default */
+    --hop-ListBox-background-color: var(--hop-neutral-surface);
+    --hop-ListBox-border-color: var(--hop-neutral-border-weak);
+    --hop-ListBox-border-radius: var(--hop-shape-rounded-md);
+    --hop-ListBox-border-size: 0.0625rem;
+    --hop-ListBox-padding-inline: var(--hop-space-inset-sm);
+    --hop-ListBox-padding-block: var(--hop-space-inset-sm);
+    --hop-ListBox-min-width: 13.6rem;
```

Comment:
Why a decimal number here, the rendered value is 217.6px

Reply:
well the min width written int he design is supposed to be 218px, which I guess is actually 13.625rem.
I don't like that it's a decimal. We could just put 13rem instead?